rotkehlchen/chain/ethereum/modules/nfts.py

```diff
@@ -312,6 +316,16 @@ def delete_price_for_nft(self, asset: Asset) -> bool:
 
         return True
 
+    def _filter_ignored_nfts(self, nfts_data: Dict[ChecksumEvmAddress, List[NFT]]) -> None:
+        """Remove ignored NFTs from NFTs data."""
+        with self.db.conn.read_ctx() as cursor:
+            ignored_nfts = {nft.identifier for nft in self.db.get_ignored_assets(cursor=cursor, only_nfts=True)}  # noqa: E501
+
+        for nfts in nfts_data.values():
+            for nft in nfts:
+                if nft.token_identifier in ignored_nfts:
+                    nfts.remove(nft)
```

Comment:
I'm not a big fun of mutability in functions but I don't know if we have a code agreement about this so is fine

Reply:
yeah yeah, the side effects & kaboom. Did a lot of redo&undo here. Will just make a copy inside the function